appveyor.yml

```diff
@@ -5,7 +5,7 @@ platform: x64 # flow needs 64b platforms
 # Install scripts. (runs after repo cloning)
 install:
   # 1. Select the right node
-  - ps: Update-NodeJsInstallation (Get-NodeJsLatestBuild $env:nodejs_version) $env:platform
+  - ps: Install-Product node $env:nodejs_version $env:platform
```

Comment:
It looks like previously Appveyor didn't have the version 12 before and we had to use the update node command to be able to use it. But now it does, so we can use the faster command now.